src/App.tsx

```diff
@@ -216,8 +239,13 @@ const App = (props: AppProps) => {
   return (
     <CyclesContext.Provider value={{ cycles, updateCycles }}>
       <ThemeContext.Provider value={{ theme, updateTheme }}>
-        <NotificationEnabledContext.Provider
-          value={{ notificationsStatus, updateNotificationsStatus }}
+        <SettingsContext.Provider
+          value={{
+            notificationsStatus,
+            updateNotificationsStatus,
+            maxDisplayedCycles,
+            updateMaxDisplayedCycles,
```

Comment:
Please use `on*` name here, because it's hard to understand what is variable and what is function